testdata/db_setup.sh

```diff
@@ -12,6 +12,8 @@ bash -x ${PWD}/testdata/docker-db-setup-scripts/docker_ps_8_0.sh
 ### Call Postgres Docker Setup Script
 bash -x ${PWD}/testdata/docker-db-setup-scripts/docker_postgres_13.sh
 
+### Call MongoDB Docker Replica Setup Script
```

Comment:
Could be removed, not needed to add as comment too

Reply:
done thanks